Blazorade.Bootstrap.Components/ProgressBar.razor.cs

```diff
@@ -0,0 +1,88 @@
+﻿using Microsoft.AspNetCore.Components;
+using Microsoft.Extensions.Logging;
+using Microsoft.JSInterop;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Blazorade.Bootstrap.Components
+{
+    /// <summary>
+    /// The ProgressBar component, when placed inside of a Progress component, defines a bootstrap progress bar 
+    /// or a section of a progress bar if multiple ProgressBar elements are used.
+    /// </summary>
+    public partial class ProgressBar
+    {
+        /// <summary>
+        /// If set to true, this progress bar will have a striped texture.
+        /// </summary>
+        [Parameter]
+        public bool IsStriped { get; set; }
+
+        /// <summary>
+        /// If set to true, any stripe will be animated.
+        /// </summary>
+        [Parameter]
+        public bool IsAnimated { get; set; }
+
+        /// <summary>
+        /// The minimum value represented by the progress bar. This must be less than both <see cref="Value"/> and <see cref="MaxValue"/>.
+        /// </summary>
+        [Parameter]
+        public double MinValue { get; set; }
+
+        /// <summary>
+        /// The maximum value represented by the progress bar. This must be greater than both <see cref="MinValue"/> and <see cref="Value"/>.
+        /// </summary>
+        [Parameter]
+        public double MaxValue { get; set; }
+
+        /// <summary>
+        /// The current value represented by the progress bar. This must fall between <see cref="MinValue"/> and <see cref="MaxValue"/>.
+        /// </summary>
+        [Parameter]
+        public double Value { get; set; }
+
+
+        protected string CalculatedWidth {
+            get
+            {
+                return $"{Math.Round(Value / MaxValue, 2) * 100}%;";
+            }
+        }
+
+        protected override void OnParametersSet()
```

Comment:
If you call the async logging methods on the base class, then you might want to change the method you override to `OnParametersSetAsync()`.

Reply:
See the other comment, but I ended up using the helper method, but calling it synchronously.